cfg/config_util_test.go

```diff
@@ -105,3 +105,25 @@ func Test_ListCacheTtlSecsToDuration_InvalidCall(t *testing.T) {
 	// Calling with invalid argument to trigger panic.
 	ListCacheTTLSecsToDuration(-3)
 }
+
+func TestIsNegativeNumber(t *testing.T) {
```

Comment:
could you please also write test case for --flag=postitive & --flag=negative number ? Basically try with = instead of " ".

Reply:
Added to main_test.go. Please take another look.